pkg/storage/etcd/etcd_helper.go

```diff
@@ -56,27 +53,24 @@ func (c *EtcdConfig) GetType() string {
 
 // implements storage.Config
 func (c *EtcdConfig) NewStorage() (storage.Interface, error) {
-	etcdClient := etcd.NewClient(c.ServerList)
-	if etcdClient == nil {
-		return nil, errors.New("Failed to create new etcd client from serverlist")
+	cfg := etcd.Config{
+		Endpoints: c.ServerList,
+		Transport: etcd.DefaultTransport,
+		// TODO: Determine if transport needs optimization
```

Comment:
agree we should check, but I want to maintain current behavior until that is resolved